### PR TITLE
test(e2e): krcil cypress tasks

### PR DESCRIPTION
**Pull request by 121watts**
Closes #218 

Bonitoo has written some e2e tests for us. Until we get forked repo credentials worked out I'm just going to copy the branches and push them up myself.

This is one of those tests.

### CODE REVIEW

File: cypress/e2e/tasks.test.ts [hunk 1]
```diff
@@ -91,6 +91,95 @@ http.post(
...
+      .then(() => {
+        cy.getByTestID('task-form-schedule-input').should(
+          'have.value',
+          '0 4 8-14 * *'
+        )
+      })
+  })
+
+  //skip until this issue is resolved
+  //https://github.com/influxdata/ui/issues/96
```
Comment (TCL735) [inline cypress/e2e/tasks.test.ts:142]: missing the spaces in the comments

File: cypress/e2e/tasks.test.ts [hunk 2]
```diff
@@ -167,6 +256,155 @@ http.post(
             })
         })
     })
+
+    //skipping until this issue is resolved
+    //https://github.com/influxdata/influxdb/issues/18478
```
Comment (TCL735) [inline cypress/e2e/tasks.test.ts:261]: same here, missing spaces

**Review by TCL735 — APPROVED** [review]

**Comment by TCL735** [comment]
a few other places have the missing space too.

**Comment by 121watts** [comment]
i will fix these once i actually get the spaces eslint rule merged in.

**Pull request merged**